yesod-form/Yesod/Form/Functions.hs

```diff
@@ -175,6 +176,23 @@ mreq :: (RenderMessage site FormMessage, HandlerSite m ~ site, MonadHandler m)
      -> MForm m (FormResult a, FieldView site)
 mreq field fs mdef = mhelper field fs mdef (\m l -> FormFailure [renderMessage m l MsgValueRequired]) FormSuccess True
```

Comment:
What do you think about rewriting this function to use `mreqMsg`?

Reply:
I think that sounds sensible. I'll try that.

Reply:
Fixed in https://github.com/yesodweb/yesod/pull/1613/commits/e5e39f353d3e60ea767f46d3b2491a95ef464dd2.